### ISSUE TITLE
Refactor SCSS/MUI to TailwindCSS - Homepage Component

### ISSUE THREAD

**Issue by mathcodes** [mathcodes/contentfordevelopers#150]
Make sure your changes have the same if not better responsiveness, colors (theme coming so look out for that addition!!!), and general style. Thanks!

**Comment by maryojo** [comment]
Hi @mathcodes, I'd like to work on this

**Comment by mathcodes** [comment]
How is everything going?!

**Comment by maryojo** [comment]
Hi @mathcodes, I'm sorry I didn't give feedback yet. The components I found that had to be refactored were very few and they were mostly components that looked like they had been replaced on the homepage, like the <ContributorSection>. Would it be possible to point out the other parts I may be missing?

**Comment by mathcodes** [comment]
Sure no problem. I would go into your local repo and search for "Material" and tons of unwanted code will show up. Here is what I got:


<img width="475" alt="image" src="https://github.com/mathcodes/contentfordevelopers/assets/17928947/b3862416-12d5-4f21-8a8e-1a6a987a78cb">

**Comment by maryojo** [comment]
Alright, I'll check this out

**Comment by maryojo** [comment]
I have checked every use of 'material' in the code and I saw that the Material UI components were not being used. About 
 6 components were defined in the pages - leetcode, hackerrank, codewars, but they were not used
<img width="475" alt="image" src="https://github.com/mathcodes/contentfordevelopers/assets/38431519/35ea163c-3262-4d48-83d4-7ee200300b7e">

Also, the ItemCard and Item used the material UI components, but there were no other components that used the ItemCard or Item component
<img width="205" alt="image" src="https://github.com/mathcodes/contentfordevelopers/assets/38431519/4dad0706-b8b0-4140-8e5a-ae3acd396242">

Am I supposed to still make changes to the ItemCard & Item components?

**Comment by mathcodes** [comment]
Great, way to knock out some spring cleaning! 

And yes, we are aiming to keep the same styles, just by using either Tailwind or raw CSS if needed. Let me know if that makes sense or if you'd like some help!

**Comment by maryojo** [comment]
Okay, I should remove unused parts and replace the styling, right?

**Comment by mathcodes** [comment]
Yup. Removing unused for sure, and when replacing with TailwindCSS (the used MUI or SASS), just do your best to keep the end result the same. Thanks! With that said, if you feel like you can improve anything, go right ahead!!! We can always discuss when I review :)
